daml-lf/spec/daml-lf-1.rst

```diff
@@ -2431,605 +2380,206 @@ exact output.
 ::
 
   Evaluation result
-    r ::= Ok v                                      -- ResOk
-       |  Err E                                     -- ResErr
-    E ::= Throw v                                   -- ErrThrow, v is a value of AnyException type
-       |  Fatal t                                   -- ErrFatal, t is a text value
-
-                           ┌──────────┐
-  Big-step evaluation      │ e  ⇓  r  │
-                           └──────────┘
-
-    —————————————————————————————————————————————————————————————————————— EvValue
-      v  ⇓  Ok v
-
-
-      e   ⇓  Err E
-    —————————————————————————————————————————————————————————————————————— EvTyAbsErasableErr
-      Λ α : ek . e  ⇓  Err E
-
-
-      e   ⇓  Ok v
-    —————————————————————————————————————————————————————————————————————— EvTyAbsErasable
-      Λ α : ek . e  ⇓  Ok (Λ α : ek . v)
 
+    r ::= Ok e                                      -- ResOk
+       |  Err err                                   -- ResErr
+    err ::=  Throw v                                -- ErrThrow, v is a value of AnyException type
+         |   Fatal t                                -- ErrFatal, t is a text value
 
-      e₁  ⇓  Err E
-    —————————————————————————————————————————————————————————————————————— EvExpAppErr1
-      e₁ e₂  ⇓  Err E
+                                              ┌──────┐
+  Applying result to an evaluation context    │ E[r] │
+                                              └──────┘
 
-      e₁  ⇓  Ok (λ x : τ . e)
-      e₂  ⇓  Err E
-    —————————————————————————————————————————————————————————————————————— EvExpAppErr2
-      e₁ e₂  ⇓  Err E
+    E[ Ok e ] = Ok E[e]
+    E[ Err err ] = Err err
 
-      e₁  ⇓  Ok (λ x : τ . e)
-      e₂  ⇓  Ok v₂
-      e[x ↦ v₂]  ⇓  r
-    —————————————————————————————————————————————————————————————————————— EvExpApp
-      e₁ e₂  ⇓  r
+                           ┌───────────┐
+  Small-step evaluation    │  e  →  r  │
+                           └───────────┘
 
-      e₁  ⇓  Err E
-    —————————————————————————————————————————————————————————————————————— EvExpTyAppErr
-      e₁ @τ  ⇓  Err E
+    —————————————————————————————————————————————————————————————————————— EvExpTyAppErasable
+      E[ (Λ α : ek . v) @τ ]  →  Ok  E[ v[α ↦ τ] ]
```

Comment:
Not related to this PR, but I do not like very much the `ek` for erasable kind. It is difficult for my brain to not associate it to expression. I have not a really better idea though.

Reply:
maybe `ke` would be better?

Reply:
Changed `ek` to `k ᵪ`, so the chi (Χ) subscript indicates erasability, like crossing out the kind.